packages/site-docs/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "blueprintjs.com/docs",
-  "version": "1.29.0",
```

Comment:
Ah weird. I thought `site-docs` and `core` moved in lockstep, but there's definitely no reason to bump `core` for this patch release. Ah well, we'll sync them up again in `1.30.0`.

Reply:
I bumped `site-docs` but not `core`. What should I do?
- [ ] Bump both `core` and `site-docs` to reflect the patch number
- [ ] Bump neither
- [ ] Leave as is

Reply:
they don't need to be in sync. what you've done in this PR is good.